plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityDomestication.java

```diff
@@ -0,0 +1,42 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.entity.AbstractHorse;
+
+public class EntityDomestication extends EntityProperty<ElementTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name domestication
+    // @input ElementTag(Number)
+    // @description
+    // Controls a horse-type entity's current domestication level.
```

Comment:
Docs should be helpful, not just echoing the tag name. What does "domestication level" mean? What value(s) are valid? What other properties are relevant to link?